### PR TITLE
Add an AWS free complete course in Arabic

### PR DESCRIPTION
**Pull request by ezz444**
this course is totally free, detailed up to 55 hours

## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-ar.md [hunk 1]
```diff
@@ -22,6 +22,7 @@
 * [SQL](#sql)
   * [PostgreSQL](#postgresql)
 * [UI/UX](#uiux)
+*  [AWS] (#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:25]:
```suggestion
* [AWS] (#aws)
```

And needs moving to line 6 to preserve alphabetization

File: books/free-programming-books-ar.md [hunk 2]
```diff
@@ -140,3 +141,6 @@
 * [مدخل إلى تجربة المستخدم](https://academy.hsoub.com/files/11-مدخل-إلى-تجربة-المستخدم-user-experience-ux/)
 
 </div>
+
+## AWS
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف] (https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:146]: Categories have headings of 3th level, and 2 blank line before and 1 after. See more reasons at linter task logs or CONTRIBUTING guidelines.

```suggestion

### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف] (https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```

Alphabetize again moving to line 34 more or less

Comment (davorpa) [inline books/free-programming-books-ar.md:146]: Categories have headings of 3th level, and 1 blank line after. See more reasons at linter task logs or CONTRIBUTING guidelines.

```suggestion

### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف] (https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```

Alphabetize again moving to line 34 more or less

**Review by davorpa — APPROVED** [review]

**Review by davorpa — CHANGES_REQUESTED** [review]
It's a good moment to apply some changes. When done, request for a review again using GitHub :arrows_clockwise: button. Meanwhile, draft status can be used to work on it if you want.

**Comment by CleanMachine1** [comment]
Considering that @ezz444 closed the PR originally I think its most likely a good idea to finish this PR making the relevant suggestions or to close it.

File: books/free-programming-books-ar.md [hunk 3]
```diff
@@ -22,6 +22,7 @@
 * [SQL](#sql)
   * [PostgreSQL](#postgresql)
 * [UI/UX](#uiux)
+* [AWS] (#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:25]:
```suggestion
* [AWS](#aws)
```

File: books/free-programming-books-ar.md [hunk 4]
```diff
@@ -140,3 +141,7 @@
 * [مدخل إلى تجربة المستخدم](https://academy.hsoub.com/files/11-مدخل-إلى-تجربة-المستخدم-user-experience-ux/)
 
 </div>
+
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف] (https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:147]:
```suggestion
* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Remove extra spaces between title and link markdown markup

File: books/free-programming-books-ar.md [hunk 5]
```diff
@@ -140,3 +141,7 @@
 * [مدخل إلى تجربة المستخدم](https://academy.hsoub.com/files/11-مدخل-إلى-تجربة-المستخدم-user-experience-ux/)
 
 </div>
+
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:147]: Add instructor / course platform

```suggestion
* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```

File: books/free-programming-books-ar.md [hunk 6]
```diff
@@ -140,3 +141,7 @@
 * [مدخل إلى تجربة المستخدم](https://academy.hsoub.com/files/11-مدخل-إلى-تجربة-المستخدم-user-experience-ux/)
 
 </div>
+
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```
Comment (davorpa) [inline books/free-programming-books-ar.md:147]:
```suggestion
```

File: books/free-programming-books-ar.md [hunk 7]
```diff
@@ -22,6 +22,7 @@
 * [SQL](#sql)
   * [PostgreSQL](#postgresql)
 * [UI/UX](#uiux)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:27]:
```suggestion

### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```

File: books/free-programming-books-ar.md [hunk 8]
```diff
@@ -22,6 +22,7 @@
...
```
Comment (davorpa) [inline books/free-programming-books-ar.md:25]:
```suggestion
* [AWS](#aws)
* [SQL](#sql)
  * [PostgreSQL](#postgresql)
* [UI/UX](#uiux)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Alphabetize attempt 1

File: books/free-programming-books-ar.md [hunk 9]
```diff
@@ -19,11 +19,15 @@
...
   * [PostgreSQL](#postgresql)
 * [UI/UX](#uiux)
 
 
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
 ### Arduino
 
 * [احترف الأردوينو](https://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) - Working Group (PDF)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:33]:
```suggestion
### Arduino

* [احترف الأردوينو](https://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) - Working Group (PDF)


### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```

File: books/free-programming-books-ar.md [hunk 10]
```diff
@@ -19,11 +19,15 @@
 * [Ruby](#ruby)
 * [Security](#security)
 * [SEO](#seo)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:22]:
```suggestion
* [AWS](#aws)
* [Ruby](#ruby)
* [Security](#security)
* [SEO](#seo)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
🚧  Alphabetize attemp 2

File: books/free-programming-books-ar.md [hunk 11]
```diff
@@ -27,6 +28,11 @@
 ### Arduino
 
 * [احترف الأردوينو](https://www.ev-center.com/uploads/2/1/2/6/21261678/arduino.pdf) - Working Group (PDF)
+
+
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
 * [اردوينو ببساطة](https://simplyarduino.com/%D9%83%D8%AA%D8%A7%D8%A8-%D8%A7%D8%B1%D8%AF%D9%88%D9%8A%D9%86%D9%88-%D8%A8%D8%A8%D8%B3%D8%A7%D8%B7%D8%A9/) - عبدالله علي عبدالله, Abdallah Ali Abdallah Elmasry (PDF)
 * [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله, Abdallah Ali Abdallah Elmasry (ODT, PDF)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:38]: Moving below Arduino resources

```suggestion
* [اردوينو ببساطة](https://simplyarduino.com/%D9%83%D8%AA%D8%A7%D8%A8-%D8%A7%D8%B1%D8%AF%D9%88%D9%8A%D9%86%D9%88-%D8%A8%D8%A8%D8%B3%D8%A7%D8%B7%D8%A9/) - عبدالله علي عبدالله, Abdallah Ali Abdallah Elmasry (PDF)
* [AVR ببساطة: من تشغيل دايود ضوئي إلى أنظمة الوقت الحقيقي](https://github.com/abdallah-ali-abdallah/Simply-AVR-Book) - عبدالله علي عبدالله, Abdallah Ali Abdallah Elmasry (ODT, PDF)


### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group

```

File: books/free-programming-books-ar.md [hunk 12]
```diff
@@ -16,6 +16,7 @@
 * [Programming in Scratch](#programming-in-scratch)
 * [Python](#python)
 * [Raspberry Pi](#raspberry-pi)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:19]: Moving near to Arduino...

```suggestion
* [AWS](#aws)
* [Programming in Scratch](#programming-in-scratch)
* [Python](#python)
* [Raspberry Pi](#raspberry-pi)
```

**Review by davorpa — CHANGES_REQUESTED** [review]
:construction: Alphabetize attempt 3

File: books/free-programming-books-ar.md [hunk 13]
```diff
@@ -13,6 +13,7 @@
 * [Operating System](#operating-systems)
 * [PHP](#php)
   * [Symfony](#symfony)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:16]:
```suggestion
* [AWS](#aws)
* [Operating System](#operating-systems)
* [PHP](#php)
  * [Symfony](#symfony)
```

File: books/free-programming-books-ar.md [hunk 14]
```diff
@@ -10,6 +10,7 @@
 * [JavaScript](#javascript)
 * [Linux](#linux)
 * [Open Source Software](#open-source-software)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:13]:
```suggestion
* [AWS](#aws)
* [JavaScript](#javascript)
* [Linux](#linux)
* [Open Source Software](#open-source-software)
```

File: books/free-programming-books-ar.md [hunk 15]
```diff
@@ -7,6 +7,7 @@
 * [C#&lrm;](#csharp)
 * [HTML/CSS](#htmlcss)
 * [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
+* [AWS](#aws)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:10]:
```suggestion
* [AWS](#aws)
* [C#&lrm;](#csharp)
* [HTML/CSS](#htmlcss)
* [Introduction to Programming in Arabic](#introduction-to-programming-in-arabic)
```

File: books/free-programming-books-ar.md [hunk 16]
```diff
@@ -31,6 +32,11 @@
...
 
 
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
+
+
 ### Artificial Intelligence
 
 * [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/) - Mohamed Lahlah (PDF)
```
Comment (davorpa) [inline books/free-programming-books-ar.md:42]:
```suggestion
### Artificial Intelligence

* [مدخل إلى الذكاء الاصطناعي وتعلم الآلة](https://academy.hsoub.com/files/17-%D9%85%D8%AF%D8%AE%D9%84-%D8%A5%D9%84%D9%89-%D8%A7%D9%84%D8%B0%D9%83%D8%A7%D8%A1-%D8%A7%D9%84%D8%A7%D8%B5%D8%B7%D9%86%D8%A7%D8%B9%D9%8A-%D9%88%D8%AA%D8%B9%D9%84%D9%85-%D8%A7%D9%84%D8%A2%D9%84%D8%A9/) - Mohamed Lahlah (PDF)


### AWS

* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```

**Review by davorpa — APPROVED** [review]

**Comment by davorpa** [comment]
> Considering that @ezz444 closed the PR originally I think its most likely a good idea to finish this PR making the relevant suggestions or to close it.

Done @CleanMachine1!!

I suppose that seeing at a glance the PR was linter faults, he/she closes it. It's a normal situation for newbies working with GitHub. I only recover it because is a interesant resource and with it we fill step by step this listing with few items

Could you review again?

**Review by CleanMachine1 — APPROVED** [review]

File: books/free-programming-books-ar.md [hunk 17]
```diff
@@ -36,6 +37,11 @@
...
 
 
+### AWS
+
+* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - AWS Riyadh User Group
```
Comment (davorpa) [inline books/free-programming-books-ar.md:42]: Add course instructor seen at first video and delegate current metadata as platform

```suggestion
* [كورس كامل بالعربى مع المهندس عيسى أبوشريف](https://www.youtube.com/playlist?list=PLOoZRfEtk6kWSM_l9xMjDh-_MJXl03-pf) - Eissa AbuSherif (AWS Riyadh User Group)
```

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

It's usually not helpful to add a category for a single resource. But I'm going to merge this, because the category headings for this list need some work, perhaps translating to arabic. Would welcome help from interested arabic users.

**Comment by AhmedElTabarani** [comment]
@davorpa, @eshellman  this is a course, not a book
We should move it to courses folder

**Comment by davorpa** [comment]
> @davorpa, @eshellman this is a course, not a book We should move it to courses folder

Go ahead then!

**Pull request merged**